routers/cart-router.js

```diff
@@ -0,0 +1,110 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+const lodash = require("lodash")
+const {extend} = lodash
+
+const { Cart } = require("../models/cart-model")
+
+router.route("/")
+.get(async (req, res) => {
+    try {
+        const cartItems = await Cart.find({})
+        res.status(200).send(cartItems)
+    } catch (err) {
+        res.status(200).json({success: false, message: "could not load cart items", errMessage: err.message})
```

Comment:
Please check status code for error. It shouldn't be 200.(in several places)

Reply:
Added that so i could see some error message on frontend, will check and change that! thanks :)